packages/jest-cli/src/watch.js

```diff
@@ -290,36 +326,6 @@ export default function watch(
         });
         startRun(globalConfig);
         break;
-      case KEYS.P:
-        testPathPatternPrompt.run(
-          testPathPattern => {
-            globalConfig = updateGlobalConfig(globalConfig, {
-              mode: 'watch',
-              testNamePattern: '',
-              testPathPattern: replacePathSepForRegex(testPathPattern),
-            });
-
-            startRun(globalConfig);
-          },
-          onCancelPatternPrompt,
-          {header: activeFilters(globalConfig)},
-        );
-        break;
-      case KEYS.T:
-        testNamePatternPrompt.run(
-          testNamePattern => {
-            globalConfig = updateGlobalConfig(globalConfig, {
-              mode: 'watch',
-              testNamePattern,
-              testPathPattern: globalConfig.testPathPattern,
-            });
-
-            startRun(globalConfig);
-          },
-          onCancelPatternPrompt,
-          {header: activeFilters(globalConfig)},
-        );
-        break;
       case KEYS.QUESTION_MARK:
         break;
```

Comment:
not for this PR, but should this `break` be here? Seems to make sense for `w` and `?` to behave the same here.